### PR TITLE
updated change-log file for v2.1.1 release

### PR DESCRIPTION
**Pull request by mshariq-nerd**

### CODE REVIEW

File: docs/changelog.md [hunk 1]
```diff
@@ -1,3 +1,16 @@
...
+- [FOGL-7291] Configuration grouping on add wizard for South, North, Notification & Filter [#245](https://github.com/fledge-iot/fledge-gui/pull/245)
+
+
+#### Fixed
+- [FOGL-7191] Fixed creating a service with " in the name causes the service to be hidden [#249](https://github.com/fledge-iot/fledge-gui/pull/249)
+- [FOGL-7271] Fixed undefined error on Control Dispatcher page [#242](https://github.com/fledge-iot/fledge-gui/pull/242)
+- [FOGL-6853] Fixed text wrapping issue for long words in syslog [#241](https://github.com/fledge-iot/fledge-gui/pull/241)
+- [FOGL-7247] Fixed Codemirror JSON/Script configuration editor width expand/shrink issues with large content [#238](https://github.com/fledge-iot/fledge-gui/pull/238)
+
+- Fixed
```
Comment (praveen-garg) [inline docs/changelog.md:13]: remove!

**Review by praveen-garg — COMMENTED** [review]
Should be based on 2.1.1RC

**Review by praveen-garg — CHANGES_REQUESTED** [review]
Invalid base.

**Comment by mshariq-nerd** [comment]
closed in favour of #258

**Pull request closed without merging**